### PR TITLE
Fetch edits

### PR DESCRIPTION
**Pull request by CodeAjaxDM**

### CODE REVIEW

**Comment by tycary** [comment]
Code looks good but I got this upon loading the creator page:
image

**Comment by tycary** [comment]
Another issue: When changing the Author Name, the content becomes unlinked from the author. What about adding an author ID?

**Comment by gracecovarrubias** [comment]
authorID sounds like a really good idea, especially if authorName is changeable.

File: views/pages/userPage/creatorPage.ejs [hunk 1]
```diff
@@ -52,134 +59,83 @@
...
+<section class="content-section">
+  <h2 class="text-center">Unpublished Content</h2>
+  <% if (authorName && unpublishedItems && unpublishedItems.length > 0) { %>
+    <div class="row content-viewer">
+      <% unpublishedItems.forEach((item, index) => { %>
+        <% if (item.ownedByAuthor) { %>
+          <div class="col-sm-3">
             <div class="thumbnail">
-              <img src="/images\beach.jpg" alt="outdoor encounters">
+              <a href="/pages/item/<%= item.itemNumber %>" class="thumbnail-link">
```
Comment (gracecovarrubias) [inline views/pages/userPage/creatorPage.ejs]: this line should be
`<a href="/item/<%= item.itemNumber %>" class="thumbnail-link">`

Comment (gracecovarrubias) [reply views/pages/userPage/creatorPage.ejs:112]: <a href="/item/<%= item.itemNumber %>" class="thumbnail-link">

**Pull request merged**